docs/setup/installation-and-configuration.md

```diff
@@ -0,0 +1,87 @@
+
+4. Fill out the form with the following values:
+
+- Name: **Mattermost MS Teams Meetings Plugin**
+- Supported account types: **Default value (Single tenant)**
+- Redirect URI: **https://(MM_SITE_URL)/plugins/com.mattermost.msteamsmeetings/oauth2/complete**. Replace `(MM_SITE_URL)` with your Mattermost server's Site URL. Select **Register** to submit the form.
+
+<img width="500 src="https://user-images.githubusercontent.com/6913320/76348298-55cd4880-62de-11ea-8e0e-4ace3a8f8fcb.png"/>
+
+5. Navigate to **Certificates & secrets** in the left pane.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/76348833-3d116280-62df-11ea-8b13-d39a0a2f2024.png"/>
+
+6. Select **New client secret > Add**, and copy the new secret on the bottom right corner of the screen. We'll use this value later in the Mattermost admin console.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/76349025-9da09f80-62df-11ea-8c8f-0b39cad4597e.png"/>
+
+7. Navigate to **API permissions** in the left pane.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/76349582-a9d92c80-62e0-11ea-9414-5efd12c09b3f.png"/>
+
+8. Select **Add a permission** and choose **Microsoft Graph** in the right pane.
+
+<img width="500" src="https://user-images.githubusercontent.com/6913320/76350226-c2961200-62e1-11ea-9080-19a9b75c2aee.png"/>
+
+9. Select **Delegated permissions**, and scroll down to select the ``OnlineMeetings.ReadWrite`` permissions.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/76350551-5a93fb80-62e2-11ea-8eb3-812735691af9.png"/>
+
+10. Select **Add permissions** to submit the form.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/80412303-abb07c80-889b-11ea-9640-7c2f264c790f.png"/>
+
+11. Select **Grant admin consent for...** to grant the permissions for the application.
+
+You're all set for configuration inside of Azure.
+
+### Step 2: Configure Plugin Settings
+
+1. Copy the **Client ID** and **Tenant ID** from the Azure portal.
+
+<img width="300" src="https://user-images.githubusercontent.com/6913320/76779336-9109c480-6781-11ea-8cde-4b79e5b2f3cd.png"/>
+
+2. Go to **System Console > PLUGINS > MS Teams Meetings**.
+3. Enter the following values in the fields provided:
+
+- `tenantID` - copy from Azure App
+- `clientID` - copy from Azure App
+- `Client Secret` - copy from Azure App (Generated in `Certificates & secrets`, earlier in these instructions)
+
+4. Choose **Save** to apply the configuration.
+
+## Onboarding users
+
+When you’ve tested the plugin and confirmed it’s working, notify your team so they get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
+
+> Hi team,
+
+> The MS Teams Meetings plugin has been configured so you can use it for calls from within Mattermost. To get started, run the `/connect` slash command from any channel within Mattermost. Visit the documentation for more information.
```

Comment:
```suggestion
> The MS Teams Meetings plugin has been configured so you can use it for calls from within Mattermost. To get started, run the `/mstmeetings connect` slash command from any channel within Mattermost. Visit the documentation for more information.
```

Reply:
What is the expected behavior after this action? And does the user need to have an MS Teams account and be logged in? If so and they're not logged in, what happens?